src/screens/SignInV2/elements/BottomOverlay/index.js

```diff
@@ -180,7 +179,7 @@ const bottomOverlayStyles = StyleSheet.create({
     lineHeight: normalize(28),
     fontSize: normalizeFontSize(22),
     fontFamily: fonts.inter[600],
-    color: COLORS.blueOnboarding,
+    color: COLORS.blue,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The color of the text has been changed from `COLORS.blueOnboarding` to `COLORS.blue`. Confirm that this change is in line with the design guidelines.
```diff
-    color: COLORS.blueOnboarding,
+    color: COLORS.blue,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->